doc/book/admin/access_control.rst

```diff
@@ -1,377 +1,1069 @@
 .. _authentication:
+.. _access_control:
 
-================================================================================
 Access control
-================================================================================
+==============
 
-This section explains how Tarantool makes it possible for administrators
-to prevent unauthorized access to the database and to certain functions.
+Tarantool enables flexible management of access to various database resources.
+The main concepts of Tarantool access control system are as follows:
 
-Briefly:
+*   A *user* is a person or program that interacts with a Tarantool instance.
+*   An *object* is an entity to which access can be granted, for example, spaces, indexes, or functions.
```

Comment:
Singular `a space`, `an index`, `a function` would be more precise, since we grant permissions per object, not for all spaces (functions) at once.